Formula/terraformer.rb

```diff
@@ -0,0 +1,32 @@
+class Terraformer < Formula
+  desc "CLI tool to generate terraform files from existing infrastructure"
+  homepage "https://github.com/GoogleCloudPlatform/terraformer"
+  url "https://github.com/GoogleCloudPlatform/terraformer.git",
+    :tag      => "0.7.4",
+    :revision => "e2dba7410c90d887918fa36eedace8c555115a47"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "on"
+
+    dir = buildpath/"src/github.com/GoogleCloudPlatform/terraformer"
+    dir.install buildpath.children
+
+    cd dir do
+      system "go", "build", "-o", bin/"terraformer"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/terraformer version")
+
+    help_output = "Available Commands"
+    assert_match help_output.to_s, shell_output("#{bin}/terraformer -h")
+
+    import_error = "google importing project aaa"
+    assert_match import_error.to_s, shell_output("#{bin}/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff")
```

Comment:
@fxcoudert I am facing similar error with this line.

```
==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer version
==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer -h
==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff
2019/06/03 07:56:03 google importing project aaa
2019/06/03 07:56:03 google importing... gcs
2019/06/03 07:56:03 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
2019/06/03 07:56:03 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
Error: terraformer: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<1>.
/Library/Ruby/Gems/2.3.0/gems/test-unit-3.1.5/lib/test/unit/assertions.rb:55:in `block in assert_block'
```

Reply:
It expected a zero exit code. Pass the expected exit code as second argument to `shell_output`.

Reply:
ok, will try it now.

Reply:
not quite working with this line
```
assert_match import_error.to_s, shell_output("#{bin}/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff", 0)

==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff
2019/06/03 09:50:14 google importing project aaa
2019/06/03 09:50:14 google importing... gcs
2019/06/03 09:50:14 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
2019/06/03 09:50:14 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
Error: terraformer: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<1>.
/Library/Ruby/Gems/2.3.0/gems/test-unit-3.1.5/lib/test/unit/assertions.rb:55:in `block in assert_block'
```

Reply:
The exit code is 1, so if you expect that, pass `1` as second argument.

Reply:
Tried that as well, not working as expected.
```
assert_match import_error.to_s, shell_output("#{bin}/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks --connect=true --zone=europe-west1-a --projects=aaa,fff", 1)

==> /usr/local/Cellar/terraformer/0.7.4/bin/terraformer import google --resources=gcs,forwardingRules,httpHealthChecks
2019/06/04 10:00:38 google importing project aaa
2019/06/04 10:00:38 google importing... gcs
2019/06/04 10:00:38 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
2019/06/04 10:00:38 google: could not find default credentials. See https://developers.google.com/accounts/docs/application-default-credentials for more information.
Error: terraformer: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: </aaa/> expected to be =~
<"">.
```

Reply:
Adopt the similar fix from helmfile, should be good now. Will squash the changes.